packstream/packstream-specification-1.md

```diff
@@ -21,7 +21,7 @@ The core data types are as follows:
 | [`Bytes`](#bytes)           | byte array                                    |
 | [`String`](#string)         | unicode text, **UTF-8**                       |
 | [`List`](#list)             | ordered collection of values                  |
-| [`Dictionary`](#dictionary) | ordered collection of key-value entries       |
```

Comment:
The dictionary is ordered

Reply:
Hmm, this contradicts the [Cypher manual](https://neo4j.com/docs/cypher-manual/current/syntax/values/#composite-types) and the [official driver source](https://github.com/neo4j/neo4j-java-driver/blob/4.1/driver/src/main/java/org/neo4j/driver/internal/messaging/v1/ValueUnpackerV1.java#L79), which uses an unordered HashMap to store the key/value pairs. Did this definition change, or is `Dictionary` meant to be different from `Map`?

Reply:
Its a bug, it should just be a set, no order guaranteed.